src/index.js

```diff
@@ -54,36 +59,44 @@ function determineValidID(event) {
   }
 }
 
+function loadTravelerInfo(userID) {
```

Comment:
Lots of missing semicolons. I know JS doesn't require but it helps readability and is typically required for linters.